supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -31,6 +31,17 @@
 
 *See also*:
 
+[discrete]
+[[weak-dependencies]]
+==== image:images/yes.png[yes] Weak dependencies (noun)
+*Description*: In Red Hat Enterprise Linux, _Weak dependencies_ are variants of the `Requires` directive that describe dependencies that are not necessary for the package to work but are considered useful by the package author.
+
+*Use it*: yes
+
+*Incorrect forms*: weak dependencies
```

Comment:
if we're clarifying here that lc w is wrong, we should do that with all other similar terms, no? (like "Source RPM" for example)

Reply:
Removing this from incorrect forms